scripts/templates.py

```diff
@@ -84,23 +86,41 @@ def read_environ_config(
     return load_list
 
 
-def read_yaml_files(yaml_dirs, name: str = "config.yaml"):
+def read_yaml_files(
+    yaml_dirs: list[str],
+    name: str | None = None,
+):
     """Read all YAML files in the given directories and return a dictionary
 
     This function will not traverse into sub-directories.
 
     yaml_dirs: list of directories to read YAML files from
+    name: (optional) name of the file(s) to read, either filename or regex. Defaults to config.yaml and default.config.yaml
     """
 
+    files = []
+    if name is None:
+        # Order is important!
+        for directory in yaml_dirs:
+            for yaml_file in Path(directory).glob("default.config.yaml"):
+                files.append(yaml_file)
+            for yaml_file in Path(directory).glob("config.yaml"):
+                files.append(yaml_file)
```

Comment:
```suggestion
        for directory in yaml_dirs:
            files.extend(Path(directory).glob("default.config.yaml"))
            files.extend(Path(directory).glob("config.yaml"))
```
I think that is the same.